h2o-genmodel/src/main/java/hex/genmodel/utils/ParseUtils.java

```diff
@@ -53,6 +53,7 @@ else if (defVal instanceof Double)
                 return Double.valueOf(input);
         }
 
+        // TODO why do we not expect primitive types as default values since arrays of primitive values are returned?
```

Comment:
do you plan on fixing this TODO?

Reply:
no, at least not in this PR. Didn't know what to do with this so put a comment to take o look here. Maybe we can create a Jira ticket with low priority? Or at least leave a comment for somebody who will try to use it to avoid confusion?

Reply:
TODO removed and FIXME comment moved to ParseUtils#tryParse() without FIXME so that it just informs how tryParse() works